deploy/operator/ztp/baremetalHost.j2

```diff
@@ -37,6 +37,9 @@ metadata:
     infraenvs.agent-install.openshift.io: '{{ infraenv_name }}'
   annotations:
     bmac.agent-install.openshift.io/hostname: '{{ host["name"] }}'
+{% if baremetalhosts_ignition_override|string != "False" %}
+    bmac.agent-install.openshift.io/ignition-config-overrides: '{{ baremetalhosts_ignition_override }}'
```

Comment:
Need to escape the JSON, otherwise it will be simply nested, I think
```suggestion
    bmac.agent-install.openshift.io/ignition-config-overrides: '{{ baremetalhosts_ignition_override | tojson }}'
```